Makefile

```diff
@@ -1,51 +1,96 @@
-.PHONY: all clean format debug release pull update
+.PHONY: all clean format debug release duckdb_debug duckdb_release pull update
 
 all: release
 
+MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+PROJ_DIR := $(dir $(MKFILE_PATH))
+
 OSX_BUILD_UNIVERSAL_FLAG=
-GENERATOR=
-ifeq ($(GEN),ninja)
-	GENERATOR=-G "Ninja"
-endif
 ifeq (${OSX_BUILD_UNIVERSAL}, 1)
 	OSX_BUILD_UNIVERSAL_FLAG=-DOSX_BUILD_UNIVERSAL=1
 endif
-BUILD_FLAGS=-DEXTENSION_STATIC_BUILD=1 -DBUILD_TPCH_EXTENSION=1 ${OSX_BUILD_UNIVERSAL_FLAG}
+ifeq (${STATIC_LIBCPP}, 1)
+	STATIC_LIBCPP=-DSTATIC_LIBCPP=TRUE
+endif
 
-DUCKDB_DIRECTORY=
-ifndef DUCKDB_DIR
-	DUCKDB_DIRECTORY=duckdb
-else
-	DUCKDB_DIRECTORY=${DUCKDB_DIR}
+ifeq ($(GEN),ninja)
+	GENERATOR=-G "Ninja"
+	FORCE_COLOR=-DFORCE_COLORED_OUTPUT=1
 endif
 
+BUILD_FLAGS=-DEXTENSION_STATIC_BUILD=1 -DBUILD_TPCH_EXTENSION=1 -DBUILD_PARQUET_EXTENSION=1 ${OSX_BUILD_UNIVERSAL_FLAG} ${STATIC_LIBCPP}
+
+CLIENT_FLAGS :=
+
+# These flags will make DuckDB build the extension
+EXTENSION_FLAGS=-DDUCKDB_OOT_EXTENSION_NAMES="sqlite_scanner" -DDUCKDB_OOT_EXTENSION_SQLITE_SCANNER_PATH="$(PROJ_DIR)" -DDUCKDB_OOT_EXTENSION_SQLITE_SCANNER_SHOULD_LINK="TRUE" -DDUCKDB_OOT_EXTENSION_SQLITE_SCANNER_INCLUDE_PATH="$(PROJ_DIR)src/include"
```

Comment:
These cmake variables have been renamed since all extensions, in-tree and out-of-tree now use them. The extension template is outdated as we changed this recently and it still points to 0.8.1. Check out https://github.com/duckdblabs/duckdb_azure how it (likely, pr in progress) will be